src/molecules/formfields/DateField/date_field.module.scss

```diff
@@ -13,23 +13,23 @@ $border: 1px solid color('neutral-4');
 }
 
 .label-wrapper {
+  display: block;
```

Comment:
I'm a little confused why this was updated, it seems to break the field when a `subLabel` is added. Can you rebase from some of the updates I made when putting all Athenaeum form fields into life-web? That might clean things up for this PR